pressbooks/pressbooks
# Add clone book page for lowly users

Issue: https://github.com/pressbooks/pressbooks/issues/1899

The clone book page now is under `admin.php?page=pb_cloner`, and is accessible to all users, including 'lowly users' since this change gives 'lowly users' the `subscriber` role on the root site.

The cloner page is now accessible for lowly users at the top menu under `My Books`.

### Testing notes
-  Create a lowly user through: `Network Admin > Users > Add new`, it's in: `wp-admin/network/user-new.php`
- Update or check the password for the new user to make sure you'll have access to log in.
- Log in with the new user and check the dashboard remains the same as any lowly user, including top and sidebar menus, widgets, etc.
- Check at the top menu in `My Books` to confirm `Clone a book` sub menu is present. You should be able to access to the page and be able to clone a book successfully.
- Once you have cloned a book, you should still be able to access the book's dashboard using the top menu under `My Books`.
- Now the user can clone additional books under `<BOOK_URL>/wp-admin/admin.php?page=pb_cloner` or `/wp-admin/admin.php?page=pb_cloner`, depending on whether the user accesses the link from the book's dashboard or the main dashboard.
